build/yamls/flow-aggregator/base/conf/flow-aggregator.conf

```diff
@@ -55,3 +40,45 @@ apiServer:
 
   # TLS min version from: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13.
   #tlsMinVersion:
+
+# flowCollectorSink contains external IPFIX or JSON collector related configuration options.
+flowCollectorSink:
+  # Enable is the switch to enable exporting flow records to external flow collector.
+  #enable: false
```

Comment:
I want to confirm that you tested the Aggregator with just Clickhouse and no external flow collector? Did it work correctly? I can't tell if the flow aggregator code depends on an external flow collector being configured or not...